pkg/cli/environment.go

```diff
@@ -230,5 +231,9 @@ func (s *EnvSettings) SetNamespace(namespace string) {
 
 // RESTClientGetter gets the kubeconfig from EnvSettings
 func (s *EnvSettings) RESTClientGetter() genericclioptions.RESTClientGetter {
+	s.config.WrapConfigFn = func(c *rest.Config) *rest.Config {
+		c.UserAgent = version.GetUserAgent()
+		return c
+	}
```

Comment:
My concern is that a `WrapConfigFn` is already setup. Won't this overwrite that? Should we combine them? See...

https://github.com/helm/helm/blob/5a8e91a010a923c3505c19954cfda4f01432bbc9/pkg/cli/environment.go#L118-L124

Reply:
Yeah, the lines mentioned by you were introduced after my original PR. Good catch! I am moving my change here